CMSgov/ato-blueprint
# Initial installation of cypress

This PR includes:
1. initial installation of Cypress
2. update to `.gitignore` to ignore the `/node_modules` and `/screenshots` directory
3. instruction on running Cypress

To test this work locally:
1. pull down code to your local machine
2. run `npm install` from project root directory (installs Cypress and required underlying libraries
3. run `npx cypress open` to start the test server and open the Cypress test runner
4. To see the tests run in real time, click "Run 20 integration tests" in the Cypress test runner app
